### DISCUSSION TITLE
Doesn't schema validation check empty string?

### DISCUSSION THREAD

**Discussion by becory** [vitalik/django-ninja#620] [Q&A] [answered]
Hi, I found the issue or not.

I create a schema
```python
class UserSchema(Schema):
    name: str
    email: str
```

I give these keys all empty values when I call API use the payload like below:
```json
{
"name": "",
"email": ""
}
```

It can pass the validation.
Is it a bug or is django-ninja not providing this check? 😕

**Comment by mom1** [comment, marked as answer]
Hello, this is normal pydantic behavior. try like this
```python
class UserSchema(Schema):
    name: str = Field(..., min_length=1)
    email: str = Field(..., min_length=1)
```

**Reply by becory** [reply]
Thank you so much.
